website/package.json

```diff
@@ -1,27 +1,20 @@
 {
   "scripts": {
-    "start": "node server/server.js",
-    "test": "node server/generate.js",
-    "gh-pages": "node publish-gh-pages.js",
+    "start": "docusaurus-start",
+    "build": "docusaurus-build",
+    "gh-pages": "docusaurus-publish",
+    "examples": "docusaurus-examples",
     "crowdin-upload": "crowdin-cli --config ../crowdin.yaml upload sources --auto-update -b master",
-    "crowdin-download": "crowdin-cli --config ../crowdin.yaml download -b master"
+    "crowdin-download": "crowdin-cli --config ../crowdin.yaml download -b master",
```

Comment:
I don't see `crowdin-download` nor `crowdin-upload` used anywhere – is this expected and somehow used implicitly? 

Also, `README.md` would an use update (e.g. port is outdated)